src/odemis/gui/cont/streams.py

```diff
@@ -206,6 +218,33 @@ def __init__(self, stream_bar, stream, tab_data_model, show_panel=True, view=Non
 
         stream_bar.add_stream_panel(self.stream_panel, show_panel)
 
+    def _display_metadata(self):
+        """ 
+        Display metadata for integration time, ebeam voltage, probe current and
+        emission/excitation wavelength
+        """
+
```

Comment:
You need to check that there is a `.raw`. It should always be present, but it might be empty. So I'd suggest to add something like `if not self.stream.raw: return`